queue/lib/txgh-queue/error_handlers/server_response.rb

```diff
@@ -0,0 +1,22 @@
+require 'txgh-server'
+
+module TxghQueue
+  module ErrorHandlers
+    class ServerResponse
+      class << self
+        def can_handle?(error_or_response)
+          error_or_response.is_a?(TxghServer::Response)
+        end
+
+        def status_for(response)
+          case response.status.to_i / 100
+            when 2
+              Status.ok
+            else
+              Status.fail
```

Comment:
this indentation is non-standard (typically case statement keywords are aligned with the initial `case`)